--- a/meson.build
+++ b/meson.build
@@ -122,7 +122,6 @@
 libxxhash = dependency('libxxhash', static: true, fallback: ['xxhash', 'xxhash_dep']).as_system()
 
 libzstd = dependency('libzstd',
-                     static: true,
                      fallback: ['zstd', 'libzstd_dep'],
                      default_options: ['bin_programs=false',
                                        'legacy_level=0',
